### PR TITLE
🤖 [ergoCubSN000] [ergoCubGazeboV1] Add configuration files for the iFeel retargeting

### PR DESCRIPTION
**Pull request by GiulioRomualdi**

### CODE REVIEW

File: src/WalkingModule/app/robots/ergoCubSN000/dcm_walking/iFeel_joint_retargeting/robotControl.ini [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+joint_velocity_cut_frequency       10.0
+
+use_wrench_filter                  0
+wrench_cut_frequency               10.0
+
+
+# if true the joint is in stiff mode if false the joint is in compliant mode
+joint_is_stiff_mode     (true, true, true,
+                         true, true, true,
+                         true, true, true, true, true, true, true,
```
Comment (mebbaid) [inline src/WalkingModule/app/robots/ergoCubSN000/dcm_walking/iFeel_joint_retargeting/robotControl.ini]: Do we want to make the shoulder joints in compliant mode be the default ?

File: src/WalkingModule/app/robots/ergoCubSN000/dcm_walking/iFeel_joint_retargeting/robotControl.ini [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                         true, true, true, true, true, true, true,
+                         true, true, true, true, true, true,
+                         true, true, true, true, true, true)
+
+# if true a good joint traking is considered mandatory
+good_tracking_required  (false, false, false
+                         true, true, true,
+                         false, false, true, true, false, false, false,
```
Comment (mebbaid) [inline src/WalkingModule/app/robots/ergoCubSN000/dcm_walking/iFeel_joint_retargeting/robotControl.ini:33]: Similar to above, do we want the last test config be the default ?

File: src/WalkingModule/app/robots/ergoCubSN000/dcm_walking/iFeel_joint_retargeting/robotControl.ini (hunk 1 above)
Comment (GiulioRomualdi) [reply src/WalkingModule/app/robots/ergoCubSN000/dcm_walking/iFeel_joint_retargeting/robotControl.ini]: That's strange on my pc the compliant mode seems to be activated for the shoulder (and git is aligned)

File: src/WalkingModule/app/robots/ergoCubSN000/dcm_walking/iFeel_joint_retargeting/robotControl.ini (hunk 2 above)
Comment (GiulioRomualdi) [reply src/WalkingModule/app/robots/ergoCubSN000/dcm_walking/iFeel_joint_retargeting/robotControl.ini:33]: Having ` false, false, true, true, false, false, false,` means that the tracking for the shoulder roll pitch and the wrist is not checked

**Review by S-Dafarra — APPROVED** [review]

**Pull request merged**